src/lib.rs

```diff
@@ -154,13 +163,15 @@ fn map_activepool_events(blk: &eth::Block, events: &mut contract::Events) {
                 .filter(|log| log.address == ACTIVEPOOL_TRACKED_CONTRACT)
                 .filter_map(|log| {
                     if let Some(event) = abi::activepool_contract::events::EtherSent::match_and_decode(log) {
+                        let convert_amount = BigInt::from_str(&event.u_amount.to_string()).unwrap_or(BigInt::from(0)).to_decimal(BigInt::from(18).to_u64());
                         return Some(contract::ActivepoolEtherSent {
```

Comment:
I noticed that the subgraph returns some ether sent entities that are empty:

```
{
            "id": "a96d683ccc58260f25c85dd8b3054e7d62fb1e7202e03bfa2aeecad6e39c33ba-116",
            "evt_index": "116",
            "evt_tx_hash": "a96d683ccc58260f25c85dd8b3054e7d62fb1e7202e03bfa2aeecad6e39c33ba",
            "u_amount": "0",
            "usd_value": "0",
            "u_to": "5dc2da72254fc303680a7d644198edfdbdf07883"
          },
```

These exist on chain, however, do you think the subgraph should be creating these entities if no ether was actually sent?